lib/ello_click/affiliate.ex

```diff
@@ -10,13 +10,21 @@ defmodule ElloClick.Affiliate do
   end
 
   defp affiliate(conn) do
-    conn.request_path
+    conn
+    |> full_url
```

Comment:
Yup, but it is generally preferred to start pipelines with a "pure" value rather then a function call. It doesn't _always_ make sense to though.